core/src/test/java/hudson/cli/handlers/ViewOptionHandlerTest.java

```diff
@@ -68,7 +70,7 @@
 
     @Before public void setUp() {
 
-        MockitoAnnotations.initMocks(this);
+        MockitoAnnotations.openMocks(this);
```

Comment:
As the [Mockito documentation](https://www.javadoc.io/doc/org.mockito/mockito-core/latest/org/mockito/MockitoAnnotations.html) describes, the recommended usage pattern for `MockitoAnnotations#openMocks` is to store the returned `AutoCloseable` into a `private` field in a method annotated with `@Before` and then call `Closeable#close()` in a method annotated with `@After`. Not doing this could potentially leak state, which might make it difficult to debug issues later on.

Did you consider doing this for our usages of `MockitoAnnotations`? Feel free to let me know if this is prohibitively difficult for some reason.

Reply:
you are right, I updated the code.